### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
In this project I  used 
-HTML
-CSS
-ran linters to check errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,43 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello microverse
 
-> Description the project.
+> In this project i set up a "hello microverse" repository.No complex coding is required for this excercise.My goal here is to master all of the tools and best practice i learned about in the previous steps.
```
Comment (vechicin) [inline README.md]: [OPTIONAL] I recommend you fix all of the typos (capitalization, spacing, etc.) in this section as it makes your README and, therefore, your project, look a little unprofessional.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,43 @@
...
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Ngufor-emmanuella**
 
-👤 **Author2**
+- GitHub: [@Ngufor-emmanuella](https://github.com/Ngufor-emmanuella)
+- Twitter: [@Nguforemmanuella](https://twitter.com/Ngufor emmauella)
+- LinkedIn: [Ngufor emmanuella](https://linkedin.com/in/Ngufor emmanuella)
```
Comment (vechicin) [inline README.md]: Kindly fix the way you are linking your Twitter and LinkedIn profiles here.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (vechicin) [inline .gitignore:1]: According to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should add a `test.md` file to your `.gitignore` file. Kindly fix this.

Screen Shot 2022-05-24 at 8 00 22
.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.headertext {
+  color: blue;
+}
```
Comment (vechicin) [inline style.css]: [OPTIONAL] It is considered bad practice to use color names. I recommend you change this color to its `rgb` or `hex` equivalent. The `rgb` code for this color is `rgb (0, 0, 255)` and the `hex` code for this color is `#0000ff`

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Ngufor-emmanuella  ✋ , 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Linters checks passing.
- [x] Good PR title/description
- [x] Good commit history.
- [x] Correct GitHub Flow.  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Ngufor-emmanuella) [reply README.md]: Thanks already arranged it

File: README.md (hunk 2 above)
Comment (Ngufor-emmanuella) [reply README.md]: Thanks already arranged it

File: .gitignore (hunk 3 above)
Comment (Ngufor-emmanuella) [reply .gitignore:1]: Thanks added it

File: style.css (hunk 4 above)
Comment (Ngufor-emmanuella) [reply style.css]: Thanks I used the rgb

**Pull request closed without merging**